scripts/checkBalances.js

```diff
@@ -1,25 +0,0 @@
-'use strict';
```

Comment:
Maybe refactor to use your new util instead? This seems to have different behavior.

Reply:
Added `checkBalance` script https://github.com/axelarnetwork/axelar-examples/pull/92/commits/0fa56c849ead23b9f7212d8412f23b4e4f16a322

```
npm run checkBalance

> axelar-examples@1.0.0 checkBalance
> node scripts/runCheckBalance

==== Print balances for testnet =====
Wallet address: 0xa411977dd24F1547065C6630E468a43275cB4d7f 

Ethereum: 0.041442241412247142 ETH
Avalanche: 1.044771168558402756 AVAX
Fantom: 19.495983779078662204 FTM
Polygon: 0.198493370035677922 MATIC
Moonbeam: 131.032343455125019057 DEV
Binance: 0.0 BNB
Aurora: 0.0 ETH
Arbitrum: 0.001 ETH
Celo: 0.0 CELO
Kava: 0.0 KAVA
```